code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -8520,6 +8520,14 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 	color = "#FFA500"
 	custom_metabolism = 0.1
 
+/datum/reagent/hardcores
```

Comment:
this isn't a good naming convention because hardcores still exist (as `reagent/muhhardcores` ID `BUSTANUT` name `Hardcores`)

Reply:
i've now renamed them to softcores as blithering suggested!